### DISCUSSION TITLE
Must have/Recommended supported features to enable for Nebula Logger in Scratch Orgs?

### DISCUSSION THREAD

**Discussion by victorng8** [jongpie/NebulaLogger#729] [Q&A] [answered]
I was thinking something like

`"features": ["EnableSetPasswordInApi", "EmpPublishRateLimit:10000", "API", "HighVolumePlatformEventAddOn", "SessionIdInLogEnabled", "EventLogFile", "DefaultWorkflowUser", "Platform Cache"],`

Anything I missed out or anything I don't need?

**Comment by jongpie** [comment, marked as answer]
Hi @victorng8 - if you just want to use Nebula Logger in your scratch org, then you don't need to add any of those features. I have not encountered a situation where someone needs to add any of those features for Nebula Logger's sake. Leaving the `features` list empty &, using the default features in scratch orgs should be enough for Nebula Logger to work. You can reference [the base scratch org definition file](https://github.com/jongpie/NebulaLogger/blob/v4.13.16/config/scratch-orgs/base-scratch-def.json) used by Nebula Logger's pipeline - no additional features have been enabled/modified (and the items listed in `settings` are just personal preference, those settings don't impact Nebula Logger's features).

But if you are deploying metadata to your scratch org with a lot of complexity, these features below might help (the rest of the features shouldn't have any impact on Nebula Logger). 
- `EmpPublishRateLimit` - could be useful if are heavily using the `empApi` in your scratch org to subscribe to Nebula Logger's platform event object `LogEntryEvent__e`
- `HighVolumePlatformEventAddOn` - could be useful if you are publishing a lot of logging data in your scratch org per day
- `Platform Cache` - could be useful for caching some queries used by Nebula Logger, but this typically is more helpful in large prod orgs/sandboxes with large data volumes. It would help in a scratch org too, but the performance difference would be very small
